src/common/object.ts

```diff
@@ -55,3 +55,6 @@ export const nameRegex = new RegExp(/^(.){1,60}$/);
 export const socialHandleRegex = new RegExp(/^@?([\w-]){1,39}$/i);
 export const handleRegex = new RegExp(/^@?[a-z0-9](\w){2,38}$/i);
 export const descriptionRegex = new RegExp(/^[\S\s]{1,250}$/);
+export const emailRegex = new RegExp(
```

Comment:
Maybe add comment about origin of the regex, so that a poor soul doesn't stumble upon this and wonder wtf is this 😂🙈